sdk/src/main/java/com/oursky/authgear/ServerException.kt

```diff
@@ -9,9 +9,9 @@ class ServerException(
     val info: JSONObject? = null
 ) : AuthgearException(message) {
     constructor(json: JSONObject) : this(
-        json.getString("name"),
-        json.getString("reason"),
-        json.getString("message"),
+        json.optString("name"),
+        json.optString("reason"),
+        json.optString("message"),
```

Comment:
But the constructor accepts a non-nullable message. What is the reason behind this change?

Reply:
Is message something must be non-null from the server?
In latte, it is possible to have authgear error without message from custom ui.

Reply:
> Is message something must be non-null from the server?

Yes.

> In latte, it is possible to have authgear error without message from custom ui.

For instance?

Reply:
Then it seems a bug in custom ui that it always didn't have message now. Maybe I fix that instead.